proposals/0052-experimental-dxil-ops.md

```diff
@@ -58,9 +58,138 @@ solution. Thus this proposal explicitly avoids addressing these issues:
  * Metadata/RDAT/PSV0/Custom lowering are out of scope for this document
 
 
+## Accepted Solution
+
+The top 16 bits of the opcode shall be used to partition the opcode into ~64k
+partitions each with ~64k opcodes. The top 16 bits of the opcode are called
+the `FeatureID` and the only valid FeatureIDs are `0x0000` and `0x8000`.
+Within a given partition, opcodes must be contiguous. When opcodes are retired
```

Comment:
> opcodes must be contiguous

I don't think this is technically true. The fact that DXC requires reserving opcodes is an oddity of DXC, but we could leave gaps, the only reason not to do that is that it has an undue cost to DXC.